x-pack/examples/map_custom_raster_source/README.md

```diff
@@ -0,0 +1,17 @@
+# mapCustomRasterSource
```

Comment:
How about renaming the plugin to something more generic that can hold other examples, maybe something like `third_party_maps_source_example`? That better aligns with folder names like `third_party_vis_lens_example`